docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -94,61 +94,90 @@ expands on what differentiates IBM, including microsimulation, from other types
 Advantages of microsimulation
 +++++++++++++++++++++++++++++
 
-.. todo::
-
-  Fill out the following sections under 'Advantages of microsimulation' subheading. 
-
 Correlation between risk exposures
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-At the individual level, risk exposures are frequently correlated. Examples include high body mass index and high fasting plasma glucose, tobacco smoking and alcohol use, and childhood height and weight.
+At the individual level, risk exposures are frequently correlated. Examples include high body mass index and high fasting plasma glucose, tobacco smoking and alcohol use, 
+and childhood height and weight. Microsimulation methods allow us to assign a risk exposure value independently to simulants, such that each one follows the desired population-
+level univariate distribution (the distribution of which comes from the GBD, in our microsimulations). In the common case of a dichotomous exposure, each simulant 
+(in the same age/sex/location group) has the same probability of exposure, which is equal to the prevalence in that population.
```

Comment:
```suggestion
level univariate distribution (which frequently comes from the GBD in our microsimulations).
```

The sentence I removed already appears on the risk correlation page, right? I think it may be a bit more in-the-weeds than we need to get here.